conduct/reporting_events.md

```diff
@@ -0,0 +1,49 @@
+# Project Jupyter Code of Conduct - Reporting Guide - Jupyter Events
+
+You can make a personal report by:
+
+**Phone**: Provided at beginning of your event. This phone number will be
+monitored for the duration of the event.
+
+**In Person**: Contact a staff member, identified by STAFF badges.
```

Comment:
This means the word "STAFF" is required on badges at meetings? Or perhaps "identified by event badges", leaving the identification word up to the event.